lib/generators/shopify_app/add_marketing_activity_extension/templates/marketing_activities_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class MarketingActivitiesController < HmacVerificationController
```

Comment:
Should this template be commented on what each action does so that the developer can have contextual information?

Reply:
Decided against, since we don't want to replace / duplicate documentation. Endpoints in this controller should make a fair bit of sense to developers who have for example, tried using a marketing activity. 

Developers should be referencing https://help.shopify.com/en/api/embedded-apps/app-extensions/shopify-admin/marketing-activities/endpoints 

Let me know if you want more context / if you think that some commenting should be included since you understand the philosophy for these tools more!